computable/XML/PropertyUnitData.xml

```diff
@@ -603,6 +603,7 @@
   <Unit property_id="71" Text="S" name="siemens" conversion="1" primary="true" UCUM="S"/>
   <Unit property_id="72" Text="T" name="Tesla" conversion="1" primary="true" UCUM="T"/>
   <Unit property_id="73" Text="lm" name="lumen" conversion="1" primary="true" UCUM="lm"/>
+  <Unit property_id="74" Text="lm/m2" name="Lux" conversion="1" primary="true" UCUM="lx"/>
```

Comment:
text should probably be "lx"?

Reply:
I don't know how exactly this file is used, I assumed on Text there is the formula or something like that. 
The information and unit I've got it from https://ucum.org/ucum at the "Table 4: SI units".
If you think it should better be Text="lx" then I will change it.

Reply:
The way I understand it, "text" is the display symbol for the units, like "m" or "°C".

Reply:
This file was developed at least in part by Sam Heard as part of the very early Archetype Editor build, and I'm not sure it was ever fully populated. I agree text is the display symbol but this is more complex in original UCUM and TBH I would inclined to refer back to the original UCUM xml definiton file or the FHIR Valueset equivalent for any 'text' , particularly as we really don't have a good way to handle translations in that file.

https://build.fhir.org/valueset-ucum-units.html

I think the job of this file shoud simply be to allocate a property value to each UCUM code. 

And I'll definitely defer to @Siljeb in terms of QA of the changes as she has been much more closely involved in updates than me.

Reply:
Thx - it was just the question re 'text' field - I'm kinda suggesting that perhaps we should drop that, or at least make it clear that is for human sense meaning only, rather than something that we try to maintain / or synchronise with the UCUM source doc.